xbmc/network/upnp/UPnPServer.cpp

```diff
@@ -278,9 +278,9 @@ CUPnPServer::Build(CFileItemPtr                  item,
             object->m_ObjectID = "0";
             object->m_ParentID = "-1";
             // root has 5 children
-            if (with_count) {
+            /*if (with_count) {
                 ((PLT_MediaContainer*)object)->m_ChildrenCount = 5;
-            }
+            }*/
```

Comment:
`with_count` is forced to `false` a few lines up (see https://github.com/Paxxi/xbmc/blob/coverity/xbmc/network/upnp/UPnPServer.cpp#L265) so this is never true.